packages/ui/src/components/va-tree-view/hooks/useTreeView.ts

```diff
@@ -87,13 +88,15 @@ const useTreeView: UseTreeViewFunc = (props, emit) => {
         nodes.forEach((node: TreeNode) => {
           if (node.disabled) { return }
 
-          if (node.children?.length) { toggleChildren(node.children) }
+          const children = getChildren(node)
```

Comment:
Can it be computed not to calc this value every time?

Reply:
Nope, I'll think about it later. 😅